lib/ravelin/client.rb

```diff
@@ -14,7 +14,7 @@ class Client
     def initialize(api_key:, api_version: 2)
```

Comment:
you will need to bump the [version.rb](https://github.com/deliveroo/ravelin-ruby/blob/master/lib/ravelin/version.rb) file to `0.1.28`. Might want to merge in the latest master into your branch first. Sorry if I caused you merge conflicts.

You will also want to update the `CHANGELOG.md` with the what the changes were for this new version.

As I also just found out, this gem in public.

Reply:
Good shout! Will do!